appveyor.yml

```diff
@@ -27,7 +27,7 @@ environment:
     TEST_DATABASE_DSN: mysql://root:Password12!@127.0.0.1:3306/test_maker
     matrix:
       - dependencies: highest
-        php_ver_target: 7.2.5
+        php_ver_target: 8.0.0
```

Comment:
Tests failing, looks like we may need to bump this to 8.0.SOMETHING that actually works, which is fine.